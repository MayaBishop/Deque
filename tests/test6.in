# testing removing and item from the deque both at the front and the back
enqueue_front 1
enqueue_back 2
print
dequeue_front
dequeue_back
print
dequeue_back
dequeue_front
