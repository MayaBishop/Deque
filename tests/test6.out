success
success
1 2
2 1
success
success
failure
failure
